src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/external_tab.html.twig

```diff
@@ -24,7 +24,7 @@
  *#}
 
 <div id="external-combination-tab" class="form-horizontal row justify-content-md-center d-none">
```

Comment:
You can remove `row justify-content-md-center` and the inside `<div class="col">`

Reply:
Nice catch, thanks!

Reply:
I think you are pleased to see all the useless divs I removed in this PR 😅

Reply:
Yes! 😄 You know I am watching 👀